apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/index.tsx

```diff
@@ -136,6 +138,7 @@ function WpcomBlockEditorNavSidebar() {
 			  );
 
 	const dismissSidebar = () => {
+		recordTracksEvent( 'calypso_editor_sidebar_dismiss' );
```

Comment:
The event should probably inside the if statement (it's only a true dismissal if `isOpen && !isClosing`), but I can't reproduce this in practice.